keps/sig-windows/2578-windows-conformance/README.md

```diff
@@ -0,0 +1,414 @@
+  - [ ] (R) Ensure GA e2e tests for meet requirements for [Conformance Tests](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/conformance-tests.md) 
+  - [ ] (R) Minimum Two Week Window for GA e2e tests to prove flake free
+- [ ] (R) Graduation criteria is in place
+  - [ ] (R) [all GA Endpoints](https://github.com/kubernetes/community/pull/1806) must be hit by [Conformance Tests](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/conformance-tests.md) 
+- [ ] (R) Production readiness review completed
+- [ ] (R) Production readiness review approved
+- [ ] "Implementation History" section is up-to-date for milestone
+- [ ] User-facing documentation has been created in [kubernetes/website], for publication to [kubernetes.io]
+- [ ] Supporting documentation—e.g., additional design documents, links to mailing list discussions/SIG meetings, relevant PRs/issues, release notes
+
+[kubernetes.io]: https://kubernetes.io/
+[kubernetes/enhancements]: https://git.k8s.io/enhancements
+[kubernetes/kubernetes]: https://git.k8s.io/kubernetes
+[kubernetes/website]: https://git.k8s.io/website
+
+## Summary
+
+In general, the ad-hoc usage of bespoke ginkgo tags is cubmbersome and makes adoption of Windows workloads on Kubernetes difficult to certify.  In addition to defining these specific definitional constructs for Windows conformance in this KEP, we also aim to implement these defintitions in a declarative, fully automated certification bundle via the sonobuoy (https://sonobuoy.io/) tool, as a quick and easy way for end-users to verify windows clusters.
+
+This KEP first abstractly defines the various categories of Windows "conformance", differentiating between core conformance from feature sets that are of common interest.
+
+Next, we define how these conformance tests can be run using *existing* Kubernetes ginkgo tags, without any additional tooling.
+
+Finally, we prescribe the implementation of a generic "windows conformance" testing tool, which can be either hand-rolled (i.e. in a bash script) or, implemented as part of a broader initiative (i.e. inside of the sonobuoy.io, which is commonly used for Kubernetes verification).
+
+The initial implementation of this KEP is mainly focused on standardizing Windows Core Conformance definitions.  Other features, such as `HostProcess`, will be refined and updated with new releases, as these APIs and their standardization evolve in the community.
+
+## Motivation
+
+Running Windows containers at scale is a requirement for many mature enterprises migrating applications Kubernetes.  The requirements of a Windows Kubelet are not comparable to a Linux Kubelet, however, because of the fact that Windows does not support the same storage, security, and privileged container model that Linux does.  We thus need an unambiguous standard in place that is meaningfull and actionable for comparing Windows Kubernetes support between vendors.
```

Comment:
comments from @jdumars @perithompson 
- need OS specifics
- need backward compatibilty stuffsss

Reply:
```suggestion
Running Windows containers at scale is a requirement for many mature enterprises migrating applications Kubernetes.  The requirements of a Windows Kubelet are not comparable to a Linux Kubelet, however, because of the fact that Windows does not support the same storage, security, and privileged container model that Linux does.  We thus need an unambiguous standard in place that is meaningful and actionable for comparing Windows Kubernetes support between vendors.
```